dbf-reader/src/main/java/org/jamel/dbf/structure/DbfHeader.java

```diff
@@ -102,4 +108,22 @@ public int getNumberOfRecords() {
     public short getRecordLength() {
         return recordLength;
     }
+
+    int getFieldIndex(String fieldName) {
+        if (fieldIndexesByNames == null) {
+            initFieldIndexesByNames();
+        }
+        if (!fieldIndexesByNames.containsKey(fieldName)) {
```

Comment:
I think this check can be eliminated. Check returned value of `HashMap.get()` method. It must be not null.